src/tflite_cpu.cpp

```diff
@@ -334,33 +325,31 @@ class MLModelServiceTFLite : public vsdk::MLModelService,
             const auto* num_threads_double = num_threads->second.get<double>();
```

Comment:
I imagine this code is right, but if we're expecting num_threads to be an integer, why are we making it a double? Wouldn't the check you changed on line 328 be unnecesssary if we'd just used `.get<int>()` here?